deploy/env/dev.yaml

```diff
@@ -4,11 +4,19 @@ registration-service:
   auth-client:
     library-url: 'https://sso.prod-preview.openshift.io/auth/js/keycloak.js'
     public-keys-url: 'https://sso.prod-preview.openshift.io/auth/realms/toolchain-public/protocol/openid-connect/certs'
-    config-raw: '{
-                  "realm": "toolchain-public",
-                  "auth-server-url": "https://sso.prod-preview.openshift.io/auth",
-                  "ssl-required": "none",
-                  "resource": "crt",
-                  "clientId": "crt",
-                  "public-client": true
-                }'
\ No newline at end of file
+    config-raw:
```

Comment:
Same here. Please do not change the raw configuration. Let's keep it as is.

Reply:
done